string fault_code #
string state # state of lower_level
int32 batt_v # battery voltage
int32 batt_c # battery current, positive when charging
int32 batt_n # battery percentage 0~100
bool sen_charger # if charger is connected
bool button_emergency # if emergency button is pushed down
bool button_power
